### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by lfrancke**

### CODE REVIEW

File: docs/modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+| Stable RHEL 8
+| `https://repo.stackable.tech/repository/rpm-release/el8`
+
+|===
+
+Add this to `/etc/yum.repos.d/stackable.repo` and make sure to pick the corrrect repository:
+
+[source]
+----
+[grafana]
```
Comment (razvan) [inline docs/modules/ROOT/pages/installation.adoc:67]: Grafana ?

**Review by razvan — APPROVED** [review]

File: docs/modules/ROOT/pages/installation.adoc (hunk 1 above)
Comment (lfrancke) [reply docs/modules/ROOT/pages/installation.adoc:67]: Ups, I'll fix that, thank you.

**Review by razvan — DISMISSED** [review]
works

**Review by razvan — APPROVED** [review]
LGTM

**Pull request merged**